### PR TITLE
feat(pagination): up to date pagination documentation

### PR DESCRIPTION
**Pull request by Geobert**
Ready for a pagination feature activated by default

Check a box in https://github.com/cobalt-org/cobalt.rs/issues/395

### CODE REVIEW

File: docs/posts.md [hunk 1]
```diff
@@ -21,11 +21,6 @@ To mark a post as draft you can either set `is_draft: true` in your
 
 ## Pagination
 
-This feature is currently experimental and need to be enabled by installing Cobalt [from
```
Comment (epage) [inline docs/posts.md]: Isn't this still true, for now?

File: docs/posts.md [hunk 2]
```diff
@@ -37,37 +32,183 @@ frontmatter. All non mandatory fields have a default value as show below:
...
+`next_index_permalink`     | String              | The relative Url of the next page in the pagination. Nil if there is no next page available.
+`first_index_permalink`    | String              | The relative Url of the first page in the pagination.
+`last_index_permalink`     | String              | The relative Url of the last page in the pagination.
+`next_index`               | Integer             | Page number of the next index
+`previous_index`           | Integer             | Page number of the previous index
+`indexes`                  | Array of Paginators | If it's a meta paginator, list of paginators
+
+#### Field `permalink_suffix`
+
+This suffix is concatenated to an internal value called `pagination_root`.
```
Comment (epage) [inline docs/posts.md]: Let's not talk about internals and just say the `permalink`

Comment (epage) [reply docs/posts.md]: Then "when the permalink has an extension, ..."

File: docs/posts.md [hunk 3]
```diff
@@ -37,37 +32,183 @@ frontmatter. All non mandatory fields have a default value as show below:
...
+gives us:
+* `/tags/index.html`
+* `/tags/my_tag/index.html`
+* `/tags/my_tag/2/index.html`
+
+
+#### Field `sort_by`
+
+We can sort the posts in the paginator with multiples criteria at the same time. By
+default we sort by `published_date` but anything that makes sense in the frontmatter can
```
Comment (epage) [inline docs/posts.md]: Proposed changing this: https://github.com/cobalt-org/cobalt.rs/pull/678

File: docs/posts.md [hunk 4]
```diff
@@ -37,37 +32,183 @@ frontmatter. All non mandatory fields have a default value as show below:
...
+{% endraw %}
+
+If `indexes` is present, we list all the tags in a tag cloud, each tag being clickable to
+access to the posts that contain the tag.
+
+If not present, we are in the paginator of a tag and list the pages contained in it.
+
+#### Field `date_index`
+
+This field is only valid with the `Dates` include. It defines on which date field we want
```
Comment (epage) [inline docs/posts.md]: "defines on which"
could just be
"defines which"

File: docs/posts.md (hunk 1 above)
Comment (Geobert) [reply docs/posts.md]: Yes it is, but I think we should merge this only when the feature is activated by default :)

Comment (Geobert) [reply docs/posts.md]: But I'll restore it for now, so we can merge now and avoid conflicts :)

**Pull request merged**